libnavigation-copilot/src/main/java/com/mapbox/navigation/copilot/SearchResultUsed.kt

```diff
@@ -0,0 +1,25 @@
+package com.mapbox.navigation.copilot
+
+import androidx.annotation.Keep
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+
+/**
+ * SearchResultUsed
+ *
+ * @property provider
+ * @property id
+ * @property name
+ * @property address
+ * @property coordinates
+ * @property routablePoint
```

Comment:
Same here, can there be more description here? Maybe, what does "Used" mean in this context. Is it, user selected or what?